### PR TITLE
#1809 - Capslock value not exposed

### PR DESCRIPTION
**Pull request by KinsonDigital**
### Purpose of this PR

**Description of feature/change:**
Fixes an issue where the CapsLock value was not exposed in the enumeration used in the GameWindow.KeyDown and KeyUp events in the KeyboardKeyEventArgs parameter.

**Which part of OpenTK does this affect:**
Input

### Testing status

Manual testing

### CODE REVIEW

File: src/OpenToolkit.Windowing.Desktop/NativeWindow.cs [hunk 1]
```diff
@@ -148,7 +148,7 @@ public string ClipboardString
             }
         }
 
-        private string _title;
+        private readonly string _title;
```
Comment (jvbsl) [inline src/OpenToolkit.Windowing.Desktop/NativeWindow.cs:151]: Please do not make these readonly. But you doing so made me notice some other problems though, these should be modified somewhere. The readonly on the _joystickStates property is fine though

File: src/OpenToolkit.Windowing.GraphicsLibraryFramework/Enums/LockKeyAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+{
+    /// <summary>
+    /// Attribute for setting caps and num lock bits on when setting the input mode.
+    /// </summary>
+    /// <seealso cref="GLFW.SetInputMode(Window*,LockKeyModAttribute,bool)"/>
+    /// <seealso cref="GLFW.GetInputMode(Window*, LockKeyModAttribute)"/>
+    public enum LockKeyModAttribute
+    {
+        /// <summary>
+        /// Specify whether the lock key bits should be sued or not.
```
Comment (leezer3) [inline src/OpenToolkit.Windowing.GraphicsLibraryFramework/Enums/LockKeyAttribute.cs:20]: Typo

File: src/OpenToolkit.Windowing.Desktop/NativeWindow.cs [hunk 3]
```diff
@@ -214,7 +214,7 @@ public unsafe bool IsFocused
             }
         }
 
-        private bool _isVisible;
+        private readonly bool _isVisible;
```
Comment (NogginBops) [inline src/OpenToolkit.Windowing.Desktop/NativeWindow.cs:217]: This shouldn't be read-only either, it's a bug that this never gets set. Similar to: #1076

Comment (KinsonDigital) [reply src/OpenToolkit.Windowing.Desktop/NativeWindow.cs]: readonly keyword removed

File: src/OpenToolkit.Windowing.GraphicsLibraryFramework/Enums/LockKeyAttribute.cs (hunk 2 above)
Comment (KinsonDigital) [reply src/OpenToolkit.Windowing.GraphicsLibraryFramework/Enums/LockKeyAttribute.cs]: spelling corrected.  good catch

File: src/OpenToolkit.Windowing.Desktop/NativeWindow.cs (hunk 1 above)
Comment (KinsonDigital) [reply src/OpenToolkit.Windowing.Desktop/NativeWindow.cs]: readonly keyword removed

**Pull request merged**